FieldKit/app/components/StationSettingsView.vue

```diff
@@ -267,6 +267,8 @@
 import routes from "../routes";
 import StationFooterTabs from './StationFooterTabs';
 import Services from '../services/services';
+import { hex_string_to_byte_wise_string } from "../lib/utilities";
```

Comment:
Just curious about why not a camel case method name

Reply:
Haha. I was in C programming mode. I'll fi xit.